compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -211,7 +194,6 @@ object Completion:
       case tpd.Select(qual, _) :: _               if qual.typeOpt.hasSimpleKind => completer.selectionCompletions(qual)
       case tpd.Select(qual, _) :: _                                             => Map.empty
       case (tree: tpd.ImportOrExport) :: _                                      => completer.directMemberCompletions(tree.expr)
```

Comment:
Is this case needed?

Reply:
Yes, those are typed Trees, and this is the head of the path. We need it to find package members